src/radical/pilot/utils/misc.py

```diff
@@ -154,8 +154,8 @@ def get_resource_config(resource: str) -> Union[None, ru.Config]:
 def get_resource_fs_url(resource: str,
                         schema  : str = None) -> Union[None, ru.Url]:
     '''
-    For the given resource label, return the contact URL of the resource's file
-    system.
+    For the given resource label, return the contact URL pointing to the root
+    of the resource's file system
```

Comment:
I may have misunderstood. Is the result of this call the value of the filesystem_endpoint configuration field or is it the equivalent of `endpoint:///`?

I thought it was the former, but the comment implies the latter.

Reply:
I added more clarification (I hope).

Reply:
The documentation still isn't clear whether this is identically equal to the `filesystem_endpoint` field (or the `endpoint:///` URL) or how it maps to other known or knowable paths or symbols.

Reply:
The next sentence says:

```
This URL is equivalent to the base ``endpoint:///`` URL available for use in a `staging_directive` once a `Session` is active.
```

Reply:
Is that true, though? I thought you said that `endpoint:///` might be equivalent to a URL that can't be represented as an `sftp://` URL (or... can you use relative paths that go to parent directories of `sftp://user@host/`?). Is the `filesystem_endpoint` translated so that the return value of this function is _always_ equivalent to `endpoint:///`?

If so, that's great! Then this documentation is correct, and I was mistaken.

Reply:
Well, the same discussion we had on the other ticket I guess.  Yes, it depends on the protocol, but it is 'true enough'.  If somebody uses an `sftp` URL, she needs to know how it is interpreted.

Reply:
> If somebody uses an sftp URL, she needs to know how it is interpreted.

What I'm looking for is some sort of note (and cross-linking) for an application programmer about how to understand `endpoint:///` in terms of other seemingly-related details, like the resource config fields and the `rp.utils.misc` functions.

The full details may be beyond the scope of this issue, but I think that it is exactly in the scope of this issue to clarify the caveats of how `endpoint:///` might point to something other than what is expected.

From the current documentation, the application programmer may expect that `endpoint:///` is equivalent to `chdir('/')` within a Task, `cd /` from the `job_manager_endpoint`, or `rs.filesystem.directory(get_resource_fs_url(...)).change_dir('/')`.

Reply:
... except I'm still not sure whether `rs.filesystem.directory(get_resource_fs_url(...)).change_dir('/')` is identically equal to `get_resource_fs_url(...)`

Reply:
Updated again - further documentation should later go to the resource configs docs.